Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2819,6 +2819,29 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.9001,
 )
 
+class UpgradeWorkflow_HeavyIon(UpgradeWorkflow):
```

Comment:
Could the name be more specific on Phase-2?

Reply:
ok, I will change the name but I thought all upgrade workflows were meant for Phase-2

Reply:
It includes Run-2, Run-3 also, see
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L12-L43

Reply:
Changed the name in d1b4049bd8dffaf205aa6e4f9c4851e06c50dc90